src/frontends/auth_logic/BUILD

```diff
@@ -0,0 +1,25 @@
+#-------------------------------------------------------------------------------
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#-------------------------------------------------------------------------------
+package(
+    default_visibility = ["//visibility:public"],
+    licenses = ["notice"],
+)
+
+filegroup(
+    name = "authlogic_grammar",
+    srcs = ["//src/frontends/auth_logic:AuthLogic.g4"],
+)
```

Comment:
It's a little non-idiomatic to use a filegroup to expose a single file. You might want to just use `exports_files` to export this single file. You can use `exports_files` to set the visibility on that file and refer to that file's dependency path directly from the package(s) to which it is exposed.